### PR TITLE
Process for maintaining Build + CI contributors list

### PR DESCRIPTION
**Pull request by b0noI**
The following issues are fixed:

* process of adding new contributors to the "Build + CI" module added
* folks who qualified are explicitly added

### CODE REVIEW

File: docs/source/community/build_ci_governance.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add New Maintainer
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:4]:
```suggestion
How to Add a New Maintainer
```

File: docs/source/community/build_ci_governance.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+-------------------------
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:5]:
```suggestion
---------------------------
```

File: docs/source/community/build_ci_governance.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+For the person to be a maintainer, a person needs to:
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:7]:
```suggestion
For the person to be a maintainer, a person needs to:

```

File: docs/source/community/build_ci_governance.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* land at least six commits to the related part of the PyTorch
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:8]:
```suggestion
* land at least six commits to the related part of the PyTorch repository
```

File: docs/source/community/build_ci_governance.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+* at least one of these commits should be in the last six months
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:9]:
```suggestion
* at least one of these commits must be submitted in the last six months
```

File: docs/source/community/build_ci_governance.rst [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+To add a qualified person to the maintainers' list, please create
+a PR that adds a person to the "people of interests" page and
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:12]:
```suggestion
a PR that adds a person to the `persons of interests <https://pytorch.org/docs/master/community/persons_of_interest.html>`__ and 
```

File: docs/source/community/build_ci_governance.rst [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+merge_rules files. Current maintainers will cast their votes of
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:13]:
```suggestion
`merge_rules <https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.json>`__ files. Current maintainers will cast their votes of
```

File: docs/source/community/build_ci_governance.rst [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+support. Decision criteria for approving the PR:
+* no earlier than two business days passed before merging (make
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:15]:
```suggestion
* Not earlier than two business days passed before merging (make
```

File: docs/source/community/build_ci_governance.rst [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  sure that the majority of the contributors have seen it)
+* PR has the correct label ("module: ci")
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:17]:
```suggestion
* PR has the correct label (`module: ci`)
```

File: docs/source/community/build_ci_governance.rst [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+* there are no objections from the current maintainers
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:18]:
```suggestion
* There are no objections from the current maintainers
```

File: docs/source/community/build_ci_governance.rst [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+* there are at least three "thumbs up" from current maintainers
```
Comment (svekars) [inline docs/source/community/build_ci_governance.rst:19]:
```suggestion
* There are at least three *thumbs up* from current maintainers
```

**Review by svekars — APPROVED** [review]
Some editorial suggestions

**Comment by malfet** [comment]
I don't think I have the authority to review any policy document, but in my humble opinion (that I think I've already voiced in another PRs of that nature): set of persons of interest for a particular submodule should be a superset of people who can approve PRs to be merged.
Also, rather than editing those lists by hand, perhaps we should have a script that generates list from github users group

Last but not least - imo we should try to keep some meta-information about the repo outside of repo sourcecode.

File: docs/source/community/build_ci_governance.rst (hunk 1 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: fixed

File: docs/source/community/build_ci_governance.rst (hunk 3 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst:7]: fixed

File: docs/source/community/build_ci_governance.rst (hunk 5 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: fixed

File: docs/source/community/build_ci_governance.rst (hunk 6 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

File: docs/source/community/build_ci_governance.rst (hunk 7 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

File: docs/source/community/build_ci_governance.rst (hunk 8 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

File: docs/source/community/build_ci_governance.rst (hunk 9 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

File: docs/source/community/build_ci_governance.rst (hunk 10 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

File: docs/source/community/build_ci_governance.rst (hunk 11 above)
Comment (b0noI) [reply docs/source/community/build_ci_governance.rst]: done

**Review by seemethere — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: docs/source/community/build_ci_governance.rst [hunk 12]
```diff
@@ -0,0 +1,20 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add a New Maintainer
+---------------------------
+
+For the person to be a maintainer, a person needs to:
+
+* land at least six commits to the related part of the PyTorch repository
+* at least one of these commits must be submitted in the last six months
```
Comment (ZainRizvi) [inline docs/source/community/build_ci_governance.rst:10]: Nits

```suggestion
* Land at least six commits to the related part of the PyTorch repository
* At least one of these commits must be submitted in the last six months
```

File: docs/source/community/build_ci_governance.rst [hunk 13]
```diff
@@ -0,0 +1,20 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add a New Maintainer
+---------------------------
+
+For the person to be a maintainer, a person needs to:
+
+* land at least six commits to the related part of the PyTorch repository
```
Comment (ZainRizvi) [inline docs/source/community/build_ci_governance.rst:9]: This wording can be a bit more generalized.  Our main gate keeper is the votes

```suggestion
* land at least six commits or significant contributions to the related part of the PyTorch repository
```

File: docs/source/community/build_ci_governance.rst [hunk 14]
```diff
@@ -0,0 +1,20 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add a New Maintainer
+---------------------------
+
+For the person to be a maintainer, a person needs to:
```
Comment (ZainRizvi) [inline docs/source/community/build_ci_governance.rst:7]: Can we include a definition of what we think a maintainer is and what their responsibilities/privileges are?  The difference between a "maintainer" and a "contributor" seems important here.

Clearly, simply making any six commits isn't enough for a person to qualify. That's why we have a voting mechanism included

File: docs/source/community/build_ci_governance.rst [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+To add a qualified person to the maintainers' list, please create
+a PR that adds a person to the `persons of interests <https://pytorch.org/docs/master/community/persons_of_interest.html>`__ page and
+`merge_rules <https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.json>`__ files. Current maintainers will cast their votes of
+support. Decision criteria for approving the PR:
+* Not earlier than two business days passed before merging (make
+  sure that the majority of the contributors have seen it)
+* PR has the correct label (`module: ci`)
+* There are no objections from the current maintainers
+* There are at least three *thumbs up* from current maintainers
```
Comment (ZainRizvi) [inline docs/source/community/build_ci_governance.rst:20]: Might be useful to add a criteria for a "maintainer" to be demoted to "emeritus".

E.g. "If at least six months go by without a maintainer making contributions to pytorch, then their status will be changed to the honorary emeritus"

Comment (ZainRizvi) [inline docs/source/community/build_ci_governance.rst:20]: What if there are *thumbs down* votes? Or if there are less than three maintainers?

```suggestion
* There are at least three net *thumbs up* from current maintainers (or all maintainers vote *thumbs up* when the module has less than 3 maintainers)
```

**Pull request closed without merging**